### PR TITLE
New Posts: Web Vulnerabilities (CompSoc), Piezoelectricity and Piezoelectric Sensors (Diode), PSLV to Mars (Piston)

### PR DESCRIPTION
**Pull request by vineeth14**
### CompSoc:
**Title:** Web Vulnerabilities.
**Description:** This article explains popular web vulnerabilities.

### Diode:
**Title:** Piezoelectricity and Piezoelectric Sensors.

### Piston:
**Title:** PSLV to Mars.

### CODE REVIEW

File: blog_src/_posts/2020-01-14-web-vulnerabilities.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  
+  ## CROSS SITE REQUEST FORGERY
+
+CSRF is an attack that tricks the victim into submitting a malicious request. It inherits the identity and privileges of the victim to perform an undesired function on the victim's behalf. For most sites, browser requests automatically include any credentials associated with the site, such as the user's session cookie, IP address, Windows domain credentials, and so forth. Therefore, if the user is currently authenticated to the site, the site will have no way to distinguish between the forged request sent by the victim and a legitimate request sent by the victim.
+
+  
+  ## CROSS SITE SCRIPTING
+
+Cross Site Scripting or XSS is a vulnerability where on user of an application can send JavaScript that is executed by the browser of another user of the same application.This is a vulnerability because JavaScript has a high degree of control over a user's web browser.That is Javascript has the ability to send more HTTP requests,modify the page,access the cookies.
```
Comment (adwait1-G) [inline blog_src/_posts/2020-01-14-web-vulnerabilities.md]: where *a* user of an application.

File: blog_src/_posts/2020-01-14-web-vulnerabilities.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+XSS is typically broken down into three categories:
+
+-   Reflected XSS
+    
+-   Stored XSS
+
```
Comment (adwait1-G) [inline blog_src/_posts/2020-01-14-web-vulnerabilities.md]: Third category missing?

File: blog_src/_posts/2020-01-14-web-vulnerabilities.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+XSS is typically broken down into three categories:
+
+-   Reflected XSS
+    
+-   Stored XSS
+    
+
+ **REFLECTED XSS**
+
+Reflected XSS is done through URL manipulation.One of the deficiencies of reflected XSS however is that it requires the victim to access the vulnerable page from an attacker controlled resource. Notice that if the data paramter, wasn't provided the exploit wouldn't work.
```
Comment (adwait1-G) [inline blog_src/_posts/2020-01-14-web-vulnerabilities.md]: Notice that if the data *parameter* wasn't provided, the exploit wouldn't work.

File: blog_src/_posts/2020-01-14-web-vulnerabilities.md (hunk 2 above)
Comment (vineeth14) [reply blog_src/_posts/2020-01-14-web-vulnerabilities.md]: I've changed the number of categories to two

**Pull request merged**